contracts/Insurance.sol

```diff
@@ -181,10 +181,10 @@ contract Insurance is IInsurance, Ownable, SafetyWithdraw {
      * @dev The target amount is 1% of the leveraged notional value of the tracer being insured.
      */
     function getPoolTarget() public view override returns (uint256) {
-        int256 target = tracer.leveragedNotionalValue() / 100;
+        uint256 target = tracer.leveragedNotionalValue() / 100;
 
         if (target > 0) {
```

Comment:
No check needed since `target`/`tracer.leveragedNotionalValue()` will always be >= 0